torch/autograd/grad_mode.py

```diff
@@ -104,6 +104,11 @@ class no_grad(_DecoratorContextManager):
         disable gradients locally see :ref:`locally-disable-grad-doc` for
         more information on how they compare.
 
+    .. note::
+        This API does not apply to :ref:`forward-mode AD <forward-mode-ad>`.
+        If you want to disable forward AD for a computation, you can unpack
```

Comment:
Looks good!
Do we want to replicate the updated message for enable_grad and set_grad_enabled below?

Reply:
Hmm, those don't have a workaround right? So I'm thinking we don't need a second line for those.

Reply:
Sure!